src/extension/browserRegistry.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import path from 'path';
+import fs from 'fs';
+import debug from 'debug';
+
+const debugLogger = debug('pw:mcp:relay:browser');
+
+export function findChromeExecutable() {
+  const lookAt: Record<'linux' | 'darwin' | 'win32', string> = {
+    'linux': '/opt/google/chrome/chrome',
```

Comment:
That does not work with msedge. Why not to use open?

Reply:
open may open firefox or safari